src/gridlayout.jl

```diff
@@ -51,6 +51,7 @@ function GridLayout(nrows::Int, ncols::Int;
         width, height, tellwidth, tellheight, halign, valign, default_rowgap, default_colgap)
 
     on(computedbboxobservable(gl)) do cbb
+        # 0.7s inference time for this anonymous function. TODO? precompile
         align_to_bbox!(gl, cbb)
```

Comment:
@timholy That is crazy, why would such a small function take so long to infer?

Reply:
It's probably really the inference time for `laign_to_bbox!` itself. If inference was good throughout, it would be precompiled when `GridLayout(Int, Int)` gets precompiled. So I marked this rather than fixing it manually. I'll take a poke at `Cthulhu.descend(GridLayout, (Int, Int))` and see if I can just fix the inference problem.

Reply:
Correction: it's due to the fact that `on` doesn't force specialization of its first argument. That's fine and how it should be.

I just pushed a couple more precompiles.